src/components/text-editor/prosemirror-adapter/prosemirror-adapter.tsx

```diff
@@ -67,6 +68,15 @@ export class ProsemirrorAdapter {
     @Event()
     private change: EventEmitter<string>;
 
+    @Watch('value')
+    protected watchValue(newValue: string) {
+        if (!this.view) {
+            return;
+        }
+
+        this.updateView(newValue);
+    }
```

Comment:
```suggestion
    @Watch('value')
    protected watchValue(newValue: string) {
        if (!this.view) {
            return;
        }

        if (this.view.state.doc.textContent === newValue) {
            return;
        }

        if (newValue === '') {
            this.updateView(newValue);
        }
    }
```

If we simply update the view each time the value is changed we get some odd behaviours. One of them is that the `spacebar` no longer works, or rather that it does work but we reset the editor view and the space is not added.

I think we need to iterate on this, but for now we need to be able to clear the text editor by passing in an empty string.